README.md

```diff
@@ -1,54 +1,266 @@
+![header-bdk-light](https://user-images.githubusercontent.com/48277920/146041957-b3fd986d-b1b3-44e3-aee8-d0171df8263d.png#gh-light-mode-only)
+
+
 # Bouvet Development Kit
 
-The Bouvet Development Kit (BDK) package allows you to easily create Mixed Reality applications for the Hololens 2.
+The Bouvet Development Kit (BDK) package allows you to easily create applications for the Hololens 2 using Unity Software.
+
+# Supported version
+
+ | [![windows](https://user-images.githubusercontent.com/48277920/145994942-b90d9c8a-60b3-444f-abbd-673434ce6096.png)](https://developer.microsoft.com/windows/downloads/windows-10-sdk) [<p align="center">Windows SDK</p>](https://developer.microsoft.com/windows/downloads/windows-10-sdk)| [![unity](https://user-images.githubusercontent.com/48277920/145994938-f3637380-5050-45b1-a35a-2054229b535e.png#gh-dark-mode-only)](https://unity3d.com/get-unity/download/archive#gh-dark-mode-only) [![unity](https://user-images.githubusercontent.com/48277920/146033376-10fa0548-866a-4933-ae98-bb7908ebce42.png#gh-light-mode-only)](https://unity3d.com/get-unity/download/archive#gh-light-mode-only)  [<p align="center">Unity 2020 LTS</p>](https://unity3d.com/get-unity/download/archive)| [![visualstudio](https://user-images.githubusercontent.com/48277920/145994943-eb8bbddc-7b97-4c0e-87d7-129181387c43.png)](http://dev.windows.com/downloads) [<p align="center">Visual Studio 2019</p>](http://dev.windows.com/downloads)| 
+| :--- | :--- | :--- | 
+
+BDK has only been tested with the following versions of Unity and Visual studio so far. There are plans to move to Unity 2021 once it enters LTS, using 2021 is at your own risk.
+
+Please refer to the [Install the tools](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/install-the-tools) page for more detailed information.
+
+# Gettings Started
+
+The [Getting Started](https://github.com/bouvet/BouvetDevelopmentKit/wiki/Getting-started#installation-via-git-in-upm) section in the wiki consist of these steps:
+
+* [Installation via Git in UPM](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/install-the-tools)
+* [Setup project settings](https://github.com/bouvet/BouvetDevelopmentKit/wiki/Getting-started#setup-project-settings)
+* [Create your scene](https://github.com/bouvet/BouvetDevelopmentKit/wiki/Getting-started#create-your-scene)
+* [Building for the HoloLens 2](https://github.com/bouvet/BouvetDevelopmentKit/wiki/Getting-started#building-for-the-hololens-2)
+
+BDK can be downloaded as a package in Unity, to learn more on how to get BDK through Unity and how to start developing apps go to our [Getting started](https://github.com/bouvet/BouvetDevelopmentKit/wiki/Getting-started) page.
+
+# Documentation
+
+All documentation can be found in the GitHub [Wiki](https://github.com/bouvet/BouvetDevelopmentKit/wiki). For more information or question you can go to [Contact Information](#contact-information).
```

Comment:
For more information or question you can go to [Contact Information](#contact-information).

Litt rar setning, kanskje 

Feel free to [contact us](#contact-information). if you have any questions.